management-app/src/api/types.ts

```diff
@@ -0,0 +1,12 @@
+export type TAPIErrorsMap = {
```

Comment:
let's discuss using T as a type name prefix?

Reply:
Are you against using prefixes for types?

Reply:
yea, it's pretty obvious that it's a type and if necessary, you can check when hovering over the element

Reply:
It isn't a big deal to add just one letter to type name, but i makes the code more visually readable. You can easily distinguish enums, types, interfaces by there prefix. Also witch this approach we increase potential variants for names. For example: we have util "sameUtil" and we can name type for it the "TSomeUtil".